openedx/core/djangoapps/zendesk_proxy/v0/views.py

```diff
@@ -0,0 +1,57 @@
+"""
+Define request handlers used by the zendesk_proxy djangoapp
+"""
+from rest_framework import status
+from rest_framework.parsers import JSONParser
+from rest_framework.response import Response
+from rest_framework.throttling import UserRateThrottle
+from rest_framework.views import APIView
+
+from openedx.core.djangoapps.zendesk_proxy.utils import create_zendesk_ticket
+
+ZENDESK_REQUESTS_PER_HOUR = 15
+
+
+class ZendeskProxyThrottle(UserRateThrottle):
+    """
+    Custom throttle rates for this particular endpoint's use case.
+    """
+    THROTTLE_RATES = {
+        'user': '{}/hour'.format(ZENDESK_REQUESTS_PER_HOUR),
+    }
+
+
+class ZendeskPassthroughView(APIView):
+    """
+    TODO better docstring
+    """
+    throttle_classes = ZendeskProxyThrottle,
+    parser_classes = JSONParser,
+
+    def post(self, request):
+        """
+        request body is expected to look like this:
+        {
+            "name": "John Q. Student",
+            "email": {
+                "from": "JohnQStudent@realemailhost.com",
+                "message": "I, John Q. Student, am having problems for the following reasons: ...",
+                "subject": "Help Request"
+            },
+            "tags": ["zendesk_help_request"]
+        }
+        """
+        try:
+            create_zendesk_ticket(
+                requester_name=request.data['name'],
+                requester_email=request.data['email']['from'],
+                subject=request.data['email']['subject'],
+                body=request.data['email']['message'],
+                tags=request.data['tags']
+            )
+        except KeyError:
+            return Response(status=status.HTTP_400_BAD_REQUEST)
```

Comment:
I should probably allow exceptions from `create_zendesk_ticket` to bubble up here, then return a 500 status code. Or do we want to use a different value in those cases? There's no guarantee that a retry would work, and we'll have hit the logging/splunk alert handler already.